src/dmd/cli.d

```diff
@@ -169,14 +169,30 @@ struct Usage
         Option("c",
             "compile only, do not link"
         ),
+        Option("check=[assert|bounds|in|invariant|out|switch][=[on|off]]",
```

Comment:
Don't want to start bikeshedding, but the duplicate `=` looks strange. Why not use `check-[assert|...]`?

Reply:
Because 1) we use `on|off` elsewhere, not `+|-` 2) I want to add a `safe` option as well 3) other options may become useful than just `on|off|safe`

Reply:
I didn't mean to remove the on/off, but to replace the first `=` with `-`, e.g. `-check-assert=on` instead of `-check=assert=on`.

Reply:
Another syntax question: is it assumed that the user will write separate `-check=assert -check=in -check=out -check=bounds` calls, or do we want to support a syntax like `-check=assert,bounds,in,out` that allows all the options to be specified in a single flag?

Reply:
Perhaps in the future. For right now, I want to maximize simplicity.